bundles/org.openhab.binding.lifx/src/main/resources/ESH-INF/thing/tile.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="lifx"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="tile">
+		<label>LIFX Tile</label>
+		<channels>
+			<channel id="color" typeId="color" />
+			<channel id="temperature" typeId="temperature" />
+			<channel id="signalstrength" typeId="system.signal-strength" />
+			<channel id="tileeffecttype" typeId="tileeffecttype" />
+			<channel id="tilemorphspeed" typeId="tilemorphspeed" />
```

Comment:
This should be a channel configuration parameter and not a channel itself (see other comment).

```suggestion
```

Reply:
How can I set the channel configuration parameter if I do not use items file, but configure my items automatically?

Reply:
You can edit these channel configuration parameters in Paper UI by clicking the pencil icon. 

So it will be similar to the color channel configuration:

<table>
  <tr>
    <td>
      <a href="https://user-images.githubusercontent.com/12213581/68070768-f03adf80-fd72-11e9-9566-6089ab5ca828.png">
        <img src="https://user-images.githubusercontent.com/12213581/68070768-f03adf80-fd72-11e9-9566-6089ab5ca828.png" height="250"/>
      </a>
    </td>
    <td>
      <a href="https://user-images.githubusercontent.com/12213581/68070769-f335d000-fd72-11e9-98d6-d7129c52e3da.png">
        <img src="https://user-images.githubusercontent.com/12213581/68070769-f335d000-fd72-11e9-98d6-d7129c52e3da.png" height="250"/>
      </a>
    </td>
  </tr>
</table>

Reply:
@wborn thanks, the channel parameters are new to me. I tried to implement that for the channel effect and there is no way I can make the edit icon appear near the channel. I have been looking all over the code for a potential reason and could not find one. Could you please take a look - the code is committed now. Thanks.

Reply:
Thanks for the update. The channel configuration parameters work for me! :-)

To get it working you probably need to remove your Tile Thing and then recreate it. It's a known issue things don't get updated when their thing type definitions are updated. There's a PR for fixing that (openhab/openhab-core#1027).

Reply:
It was not enough to recreate the thing, but restarting the service helped. Now I can see it.